### DISCUSSION TITLE
Migrating from Gitlab to GitHub Enterprise cloud

### DISCUSSION THREAD

**Discussion by ryan-d-p-ubana** [community/community#128256] [Enterprise] [answered]
### Select Topic Area

Question

### Body

Hi team, we are planning to migrate data from Gitlab to Github Enterprise cloud using manual steps (git clone and git push not using the migrating repository (link). I want to confirm what are the items that will be migrated such issues, merge request, branches?

**Comment by queenofcorgis** [comment, marked as answer]
Hello @ryan-d-p-ubana, 

Using git clone and git push will import the contents of the repo, including history, and nothing else. For more historical information like that I would recommend [the migration path from the docs](https://docs.github.com/en/enterprise-cloud@latest/migrations/overview/migration-paths-to-github#gitlab-to-githubcom).

**Reply by farazladla301** [reply]
128256

**Reply by LukeLarge** [reply]
Sounds good. Thank you for information

**Comment by ryan-d-p-ubana** [comment]
Hi @queenofcorgis ,
How can I get a copy of GitHub Enterprise Importer? I've tried to use git clone --bare ... and git push --mirror. It only migrated repositories, branches and tags.

**Reply by queenofcorgis** [reply]
@ryan-d-p-ubana Enterprise Cloud Importer is only available for expert-led migrations. 

More information: [GitHub Expert Services](https://github.com/services/)

**Comment by Mondjoe** [comment]
that's mine , because i be hurry and need to learn more

**Comment by Mondjoe** [comment]
may be i need to discus with personal advise
